atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockRequestBatcherProviders.java

```diff
@@ -34,4 +35,6 @@ public interface TimeLockRequestBatcherProviders {
     TimeLockRequestBatcherProvider<MultiClientTimeLockUnlocker> unlock();
 
     TimeLockRequestBatcherProvider<MultiClientTimestampLeaseAcquirer> acquireTimestampLease();
+
+    TimeLockRequestBatcherProvider<MultiClientMinLeasedTimestampGetter> minLeasedTimestamp();
```

Comment:
This is a break. I will own fixing it in the only service using this (internal skiing product).